src/ssat/core/SsatSolver.cc

```diff
@@ -143,6 +143,9 @@ SsatSolver::parse_SDIMACS( gzFile & input_stream )
        else if ( *in == 'r' ) { // r p x_i  <-- randomly quantified variable x_i w.p. p 
           ++in;
           parsed_prob = parseFloat(in);
+          if (parsed_prob > 1.0) {
+             fprintf( stderr , "PARSE ERROR! Probability for a random quantifier not in range of [0, 1], got %lf\n", parsed_prob ) , exit(3);
```

Comment:
Why do you use `exit(3)`?

Reply:
Good question, I think any non-zero value should be enough to indicate there's an error.
Using `3` is to be consistent with the original codebase (there are many `exit(3)`s there, which I'm not sure if there's a special meaning.

Reply:
I see. Then let's keep it as it is now.